src/datasource.ts

```diff
@@ -381,7 +381,9 @@ const recordSchema = (columnName: string, schema: any, result: AdxColumnSchema[]
   }
 
   for (const name of Object.keys(schema)) {
-    const key = `${columnName}.${name}`;
```

Comment:
A columns can contain `.` in their names so in order to be able to identify dynamic columns just with the name, I have replaced it with `>`